lib/LANraragi/Utils/Constants.pm

```diff
@@ -0,0 +1,20 @@
+package LANraragi::Utils::Constants;
+
+use strict;
+use warnings;
+use utf8;
+no warnings 'experimental';
+
+use Exporter 'import';
+our @EXPORT_OK = qw(%TANK_METADATA);
+
+
+our %TANK_METADATA = (
```

Comment:
`TANK_METADATA` is only used in Model/Tankoubon.pm -- I'm not convinced it needs a spare class atm tbh

Reply:
Yeah, it's not for the present, but for the future, if any other model wants to eventually get the tanks to retrieve a piece of metadata, it's better to be able to do it by the name of the field, instead of guessing the number. Still I'm open to remove it and harcode the scores if you are still not convinced.

Reply:
Oh I meant moving the hash into Model/Tank, I do think being able to put a name onto them is good

Reply:
Yeah, I was talking about that too, since having a dedicated file for all shared constants is usually good practice, but we are not using this a lot in this project, so I guess putting it into Model/Tank is ok.